spec/features/characters/new_spec.rb

```diff
@@ -50,7 +69,62 @@
     end
   end
 
-  scenario "Creating character with icon, description and extant template"
+  scenario "Creating character with icon, description and extant template", js: true do
+    user = login
+    icon = create(:icon, user: user, keyword: 'Example icon')
+    template = create(:template, user: user, name: 'Example template')
```

Comment:
Can we make another template to be sure it selected this intentionally rather than randomly?